src/scripts/app.jsx

```diff
@@ -12,20 +12,28 @@ import "../styles/main.sass";
 const auth = resoundAPI.auth;
 
 class Root extends React.Component {
+  redirectIfLoggedOut() {
+    if (!auth.isAuthenticated()) {
+      auth.login();
+    }
+  }
+
   render() {
     return (
       <div>
-        <Route render={history => <Header auth={auth} history={history} />} />
+        <Route
+          render={history =>
+            <Header
+              auth={auth}
+              history={history}
+              onEnter={this.redirectIfLoggedOut(this)}
```

Comment:
Because of this annoyance with react and es6 not providing autobinding (https://facebook.github.io/react/blog/2015/01/27/react-v0.13.0-beta-1.html#autobinding).  There's actually a better fix to this here if you're curious: https://github.com/ProjectResound/store-upload/pull/7